### PR TITLE
Clean up trust policy with two External Ids in "Create Cluster" Doc

### PR DESCRIPTION
**Pull request by paolaperaza**
Per Slack conversation here: https://astronomer.slack.com/archives/C01UA03TPV1/p1637601248314100

Simplifying our example Trust Policy in the "Create Cluster" doc.

### CODE REVIEW

File: docs/create-cluster.md [hunk 1]
```diff
@@ -57,48 +57,39 @@ If you only have 1 existing Cluster, you will see the following under **Policy D
...
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::406882777402:root"
+      },
+      "Action": "sts:AssumeRole",
+      "Condition": {
+        "StringEquals": {
+          "sts:ExternalId": [
+            <External-ID-1>,
+            <External-ID-2>,
```
Comment (chomipi88) [inline docs/create-cluster.md:86]: @paolaperaza @cmarteepants should the comma after the second external ID be there?

Comment (cmarteepants) [reply docs/create-cluster.md:86]: No, it should not. Great catch!

Comment (paolaperaza) [inline docs/create-cluster.md:86]:
```suggestion
            <External-ID-2>
```

Comment (paolaperaza) [reply docs/create-cluster.md:86]: Oh man! Good eye @chomipi88 - removed.

**Review by chomipi88 — APPROVED** [review]

**Review by cmarteepants — CHANGES_REQUESTED** [review]
Small nit, but I would add quotes around the external-id references, since we're doing that above.

**Pull request merged**